net-p2p/nicotine+/nicotine+-3.3.6.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..13} )
+DISTUTILS_USE_PEP517=setuptools
+
+inherit distutils-r1 optfeature xdg
+
+DESCRIPTION="Graphical client for the Soulseek peer to peer network written in Python"
+HOMEPAGE="https://nicotine-plus.org/"
+SRC_URI="https://github.com/Nicotine-Plus/nicotine-plus/archive/${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/nicotine-plus-${PV}"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+# NOTE: good link - https://github.com/nicotine-plus/nicotine-plus/blob/master/doc/DEPENDENCIES.md
+BDEPEND="
+	sys-devel/gettext
+	test? (
+		dev-python/pytest
```

Comment:
Small nitpick: this dependency is handled by `distutils_enable_tests pytest` below.